documents/policies/external/Competition Requirements.md

```diff
@@ -73,25 +73,26 @@ The WCA Delegate must submit the following information when requesting approval
 2. Location(s)
    1. A location consists of the country, city, and venue.
    2. The location should also include subregions of the country that are common to include with addresses (e.g. the state a U.S. city is located in).
-   3. Competitions will be accepted regarding proximity if there are no other competitions within 100 km driving distance and less than 19 days away.
+   3. Competitions will be accepted regarding proximity if there are no other competitions within 10 km driving distance and less than 19 days away.
       1. Otherwise, the WCA Competition Announcement Team should review the circumstances of the proposed competition. The WCA Delegate requesting approval for such a competition must provide further information on why the competition should still be accepted. Possible arguments that support such a competition being accepted include:
          1. The two competitions don’t have events in common.
          2. The two competitions take place in different countries.
    4. 3x3x3 Fewest Moves simultaneous competitions
       1. 3x3x3 Fewest Moves simultaneous competitions are WCA competitions having only the 3x3x3 Fewest Moves event and taking place simultaneously in different locations.
          1. Any location must be announced at least 28 days (four weeks) before 00:00 UTC of the start date of the competition on the WCA website.
-         2. If any location of such competitions is closer than 100 km to any other competition, then
+         2. If any location of such competitions is closer than 10 km to any other competition, then
             1. both competitions should be at least 5 days apart, or
             2. the other competition must not have the 3x3x3 Fewest Moves event.
       2. A 3x3x3 Fewest Moves simultaneous competition cannot be part of any other regular competition.
          1. Time frames when both are held cannot overlap.
    5. Series competitions
-      1. Series competitions are a special classification that can be assigned to WCA Competitions that are held less than 33 days apart for which the distance between them is less than 100 km, with the intention to accommodate more competitors to compete in one area.
+      1. Series competitions are a special classification that can be assigned to WCA Competitions that are held less than 33 days apart for which the distance between them is less than 200 km, with the intention to accommodate more competitors to compete in one area.
          1. Series competitions may be accepted if the need for competitions in an area is so great that the community will benefit from having multiple competitions held in a short period of time. 
          2. All competitions within the same series must be submitted and announced together.
+            1. A series competition may have an additional competition announced at a later time if it has identical events to another competition in the series, and if it still fulfills all other requirements for a regular competition.
```

Comment:
nit to match other uses.
```suggestion
            1. Series competitions may have an additional competition announced at a later time if it has identical events to another competition in the series, and if it still fulfills all other requirements for a regular competition.
```

Reply:
Good catch! Although upon re-reading "A series may have..." is probably more grammatically correct. Do you have thoughts on that?

Reply:
I agree, but the issue is probably that we define "Series Competitions" instead of "Series." and it may be better to refactor that whole part:

5. Series ~~competitions~~
1. Series ~~competitions~~ are a special classification that can be assigned to WCA Competitions that are held less than 33 days apart for which the distance between them is less than 200 km, with the intention to accommodate more competitors to compete in one area.
 1. Series ~~competitions~~ may be accepted if the need for competitions in an area is so great that the community will benefit from having multiple competitions held in a short period of time. 
2. All competitions within the same series ~~must~~ *should* be submitted and announced together.
1. A series ~~competition~~ may have an additional competition announced at a later time if it has identical events to another competition in the series, and if it still fulfills all other requirements for a regular competition.
2. *Competitions in a* series ~~competitions~~ must clearly and publicly ~~state their condition of series competitions~~ *show that they are part of a series* and *state* their special participation terms.
1. A competitor is only permitted to participate in one *competition* of ~~the~~ *a* series ~~competitions~~.
1. If a competitor competes at more than one ~~of the involved series~~ competition~~s~~ *in a series*, only the results from the first competition will be recognized, and all other results will be removed. At the discretion of the WCA Delegate, results from all competitions in the series may be removed, and they may face disciplinary action.
2. A competition may only be part of one series.

Reply:
Fixed!